arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -0,0 +1,974 @@
+    },
+    "NamespaceCreateOrUpdateParameters": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Gets or sets Namespace data center location."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets Namespace tags."
+        },
+        "properties": {
+          "$ref": "#/definitions/NamespaceProperties",
+          "description": "Gets or sets properties of the Namespace."
+        }
+      },
+      "required": [
+        "location",
+        "properties"
+      ],
+      "description": "Parameters supplied to the CreateOrUpdate Namespace operation."
+    },
+    "NamespaceResource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Gets or sets the id of the created Namespace."
+        },
+        "location": {
+          "type": "string",
+          "description": "Gets or sets datacenter location of the Namespace."
+        },
+        "kind": {
+          "type": "string",
+          "description": "Gets or sets kind of the Namespace."
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets or sets name of the Namespace."
+        },
+        "type": {
+          "type": "string",
+          "description": "Gets or sets resource type of the Namespace."
+        },
+        "sku": {
+          "$ref": "#/definitions/ARMSku",
+          "description": "Gets or sets sku of the Namespace."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets tags of the Namespace."
+        },
+        "properties": {
+          "$ref": "#/definitions/NamespaceProperties",
+          "description": "Gets or sets properties of the Namespace."
+        }
+      },
+      "description": "Description of a Namespace resource."
+    },
+    "ARMSku": {
+      "type": "object",
+      "properties": {
+        "Name": { "type": "string" },
+        "Tier": { "type": "string" },
+        "Size": { "type": "string" },
+        "Family": { "type": "string" },
+        "Capacity": {
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "SharedAccessAuthorizationRuleProperties": {
+      "properties": {
+        "primaryKey": {
+          "type": "string",
+          "description": "The primary key that was used."
+        },
+        "secondaryKey": {
+          "type": "string",
+          "description": "The secondary key that was used."
+        },
+        "keyName": {
+          "type": "string",
+          "description": "The name of the key that was used."
+        },
+        "claimType": {
+          "type": "string",
+          "description": "The type of the claim."
```

Comment:
What could be the possible values? Is it possible to enumerate? If so can this be an expandable or non expandable enum based on your requirement?